src/Jackett.Common/Indexers/DivxTotal.cs

```diff
@@ -0,0 +1,333 @@
+            var base64EncodedData = onclick.Split('\'')[3];
+            var base64EncodedBytes = Convert.FromBase64String(base64EncodedData);
+            return Encoding.UTF8.GetString(base64EncodedBytes);
+        }
+
+        private bool CheckTitleMatchWords(string queryStr, string title)
+        {
+            // this code split the words, remove words with 2 letters or less, remove accents and lowercase
+            MatchCollection queryMatches = Regex.Matches(queryStr, @"\b[\w']*\b");
+            var queryWords = from m in queryMatches.Cast<Match>()
+                where !string.IsNullOrEmpty(m.Value) && m.Value.Length > 2
+                select Encoding.UTF8.GetString(Encoding.GetEncoding("ISO-8859-8").GetBytes(m.Value.ToLower()));
+
+            MatchCollection titleMatches = Regex.Matches(title, @"\b[\w']*\b");
+            var titleWords = from m in titleMatches.Cast<Match>()
+                where !string.IsNullOrEmpty(m.Value) && m.Value.Length > 2
+                select Encoding.UTF8.GetString(Encoding.GetEncoding("ISO-8859-8").GetBytes(m.Value.ToLower()));
+            titleWords = titleWords.ToArray();
+
+            foreach (var word in queryWords)
+            {
+                if (!titleWords.Contains(word))
+                    return false;
+            }
+
+            return true;
+        }
+
+        private string TryToCleanSeriesTitle(string title, string episodeTitle)
+        {
+            // title = Superman
+            // episodeTitle = Superman1x12
+            var newTitle = episodeTitle;
+            try
+            {
+                newTitle = newTitle.Replace(title, title + " ");
+                Regex r = new Regex("(([0-9]+)x([0-9]+))", RegexOptions.IgnoreCase);
+                Match m = r.Match(newTitle);
+                if (m.Success)
+                {
+                    var season = "S" + m.Groups[2].Value.PadLeft(2, '0');
+                    var episode = "E" + m.Groups[3].Value.PadLeft(2, '0');
+                    newTitle = newTitle.Replace(m.Groups[1].Value, season + episode);
+                }
+                newTitle = newTitle.Replace(" COMPLETA", "").Replace(" FINAL TEMPORADA", "");
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(e);
+                throw;
+            }
+            // return Superman S01E012
+            return newTitle;
+        }
+
+        private DateTime TryToParseDate(string dateToParse, DateTime dateDefault)
+        {
+            var date = dateDefault;
+            try
+            {
+                date = DateTime.ParseExact(dateToParse, "dd-MM-yyyy", CultureInfo.InvariantCulture);
+            }
+            catch
+            {
+                // ignored
+            }
+            return date;
+        }
+
+        private long TryToParseSize(string sizeToParse, long sizeDefault)
+        {
+            var size = sizeDefault;
```

Comment:
searching for `mandalorian S01E08` does not appear to work?
![image](https://user-images.githubusercontent.com/6933240/71842599-e0e76200-3126-11ea-8f01-64bcff312669.png)

![image](https://user-images.githubusercontent.com/6933240/71842792-5bb07d00-3127-11ea-96fa-5892a9e20375.png)

Reply:
This is not supported by the site. I can do a few tricks to make it work with Sonarr, but I prefer to do it in the future. It will take time...